hw4_sn.rb

```diff
@@ -0,0 +1,25 @@
+puts -123.odd?
+puts 6893.next
+puts 0.pred
+puts (12.to_f / 4.5).round
+puts ((28%6)*4.7).floor
+puts a1="Brand new bicycle\r\n".gsub("\r","").gsub("\n","")
+puts a2="Brand new bicycle".split(" ").delete_at(1)
+
+var1= "Brand new bicycle"
+var1= var1.split(" ")
+var1.delete(var1.last)
+puts a3=var1.join(" ")
```

Comment:
All good, but FYI you can also write this in one line:
"Brand new bicycle".split(" ")[0..-2].join(" ")